Learn-Information-Security-by-Building-a-Contact-Form/server070.js

```diff
@@ -0,0 +1,20 @@
+const express = require("express");
+const app = express();
+const helmet = require("helmet");
+// https://helmetjs.github.io/ - for more information on all packages and uses.
+
+app.use(helmet.dnsPrefetchControl());
+app.use(helmet.hidePoweredBy());
+app.use(helmet.ieNoOpen());
+app.use(helmet.noSniff());
+app.use(helmet.xssFilter());
+app.use(helmet.frameguard());
+app.use(
```

Comment:
Walk learners through calling `helmet.hsts({ maxAge: 5184000 /* 60 days in seconds */ })` and why.